docs/modules/release-notes/pages/5-3-0.adoc

```diff
@@ -0,0 +1,226 @@
+
+=== Storage
+
+* Disk tier option of Tiered Storage is now disabled by default.
+https://github.com/hazelcast/hazelcast/pull/23747[#23747]
+
+=== Distribution
+
+* Hazelcast no longer depends on JRuby; the JRuby `ScriptFactory` dependency must be explicitly added to the application.
+https://github.com/hazelcast/hazelcast/pull/23355[#23355]
+* Added the Kafka Connect extension to the distribution.
+https://github.com/hazelcast/hazelcast/pull/23312[#23312]
+* Shaded dependencies for Hazelcast Platform have been combined in a dedicated package (`com.hazelcast.shaded`).
+https://github.com/hazelcast/hazelcast/pull/23124[#23124]
+
+=== Networking
+
+* Added socket options for per-socket keep-alive configuration: `keep-count`, `keep-idle-seconds`, and `keep-interval-seconds`.
+You can set these options using either the advanced network configurations or Hazelcast system properties.
+See xref:clusters:network-configuration.adoc#configuring-tcp-keep-alive[Configuring TCP Keep-Alive].
+https://github.com/hazelcast/hazelcast/pull/23816[#23816], https://github.com/hazelcast/hazelcast/pull/24050[#24050]
+
+=== SQL Engine
+
+* The `CREATE MAPPING` statement for Compact Serialization now supports nullable primitives.
+https://github.com/hazelcast/hazelcast/pull/24300[#24300]
+* Implemented the `GET_DDL` function.
+https://github.com/hazelcast/hazelcast/pull/23595[#23595]
+* Implemented the `SHOW DATA CONNECTIONS` statement to show the connections created by the `CREATE CONNECTIONS` statement.
+https://github.com/hazelcast/hazelcast/pull/24028[#24028]
+* Added support of providing declarative partitioning strategies based on attributes. These attributes represent specific partition keys for a map.
+You can also configure this strategy dynamically.
+https://github.com/hazelcast/hazelcast/pull/23535[#23535], https://github.com/hazelcast/hazelcast/pull/23954[#23954]
+* Improved the nested loop joins algorithm by making the `CreateDagVisitor` interface generic.
+https://github.com/hazelcast/hazelcast/pull/23368[#23368]
+* Implemented the `TO_CHAR` function which is capable of formatting dates, timestamps, and numbers.
+https://github.com/hazelcast/hazelcast/pull/22529[#22529]
+* Implemented `JSON=VARCHAR` comparison.
+https://github.com/hazelcast/hazelcast/pull/22746[#22746]
+
+=== Jet Engine
+
+* Each Jet engine job was creating a client for connecting to the remote cluster, resulting in an excessive number of clients.
+This has been fixed by introducing a single data store which can be reused for all job instances.
+https://github.com/hazelcast/hazelcast/pull/23472[#23472], https://github.com/hazelcast/hazelcast/pull/23849[#23849]
+* Added MongoDB read (source) and write (sink) processors.
+https://github.com/hazelcast/hazelcast/pull/23348[#23348]
+* Added new messages (`uploadJobMetaData` and `uploadJobMultipart`) to the binary client protocol to be able to
+execute non-Java codes for streaming data pipelines, i.e., to submit Jet jobs.
```

Comment:
This is not public API, not sure if it should be in RN.

Reply:
Removed